src/main/java/frc/trigon/robot/subsystems/climber/ClimberConstants.java

```diff
@@ -0,0 +1,182 @@
+import frc.trigon.robot.constants.RobotConstants;
+import org.trigon.hardware.RobotHardwareStats;
+import org.trigon.hardware.phoenix6.talonfx.TalonFXMotor;
+import org.trigon.hardware.phoenix6.talonfx.TalonFXSignal;
+import org.trigon.hardware.simulation.SimpleMotorSimulation;
+
+public class ClimberConstants {
+    private static final int
+            RIGHT_MOTOR_ID = 14,
+            LEFT_MOTOR_ID = 15;
+    private static final String
+            RIGHT_MOTOR_NAME = "RightClimberMotor",
+            LEFT_MOTOR_NAME = "LeftClimberMotor";
+    static final TalonFXMotor
+            RIGHT_MOTOR = new TalonFXMotor(
+            RIGHT_MOTOR_ID,
+            RIGHT_MOTOR_NAME,
+            RobotConstants.CANIVORE_NAME
+    ),
+            LEFT_MOTOR = new TalonFXMotor(
+                    LEFT_MOTOR_ID,
+                    LEFT_MOTOR_NAME,
+                    RobotConstants.CANIVORE_NAME
+            );
+
+    private static final InvertedValue
+            RIGHT_MOTOR_INVERTED_VALUE = InvertedValue.Clockwise_Positive,
+            LEFT_MOTOR_INVERTED_VALUE = InvertedValue.Clockwise_Positive;
+    private static final NeutralModeValue NEUTRAL_MODE_VALUE = NeutralModeValue.Brake;
+    static final boolean ENABLE_FOC = true;
+    static final double //TODO: calibrate
+            MAX_NON_CLIMBING_VELOCITY = RobotHardwareStats.isSimulation() ? 1 : 1,
+            MAX_NON_CLIMBING_ACCELERATION = RobotHardwareStats.isSimulation() ? 1 : 1,
+            MAX_CLIMBING_VELOCITY = RobotHardwareStats.isSimulation() ? 1 : 1,
+            MAX_CLIMBING_ACCELERATION = RobotHardwareStats.isSimulation() ? 1 : 1;
+    private static final double //TODO: calibrate
+            GROUNDED_P = RobotHardwareStats.isSimulation() ? 500 : 0,
+            GROUNDED_I = RobotHardwareStats.isSimulation() ? 0 : 0,
+            GROUNDED_D = RobotHardwareStats.isSimulation() ? 0 : 0,
+            GROUNDED_KS = RobotHardwareStats.isSimulation() ? 0.0046109 : 0,
+            GROUNDED_KV = RobotHardwareStats.isSimulation() ? 8.7858 : 0,
+            GROUNDED_KA = RobotHardwareStats.isSimulation() ? 0.17776 : 0;
+    static final double
+            GROUNDED_A = 0,
+            GROUNDED_B = 0,
+            GROUNDED_C = 0;
+    private static final double //TODO: calibrate
+            ON_CHAIN_P = RobotHardwareStats.isSimulation() ? 500 : 0,
+            ON_CHAIN_I = RobotHardwareStats.isSimulation() ? 0 : 0,
+            ON_CHAIN_D = RobotHardwareStats.isSimulation() ? 0 : 0,
+            ON_CHAIN_KS = RobotHardwareStats.isSimulation() ? 0.0046109 : 0,
+            ON_CHAIN_KV = RobotHardwareStats.isSimulation() ? 8.7858 : 0,
+            ON_CHAIN_KA = RobotHardwareStats.isSimulation() ? 0.17776 : 0;
+    static final double
+            ON_CHAIN_A = 0,
+            ON_CHAIN_B = 0,
+            ON_CHAIN_C = 0;
+    static final int
+            GROUNDED_SLOT = 0,
+            ON_CHAIN_SLOT = 1;
+    static final double GEAR_RATIO = 74.67;
```

Comment:
If it's a repeating number, make a lot more 6s and then put a seven as the last possible digit

Reply:
this is the number it says in the spreadsheet